demisto_sdk/commands/common/content/objects/pack_objects/lists/lists.py

```diff
@@ -0,0 +1,28 @@
+from typing import Union
+
+import demisto_client
+from wcmatch.pathlib import Path
+
+from demisto_sdk.commands.common.constants import LISTS, FileType
+from demisto_sdk.commands.common.content.objects.pack_objects.abstract_pack_objects.json_content_object import \
+    JSONContentObject
+
+
+class Lists(JSONContentObject):
```

Comment:
why is the name in plural? this object reffer to a single item

Reply:
The class `List()` was shadowing the builtin python class, and caused errors while running, so I thought `Lists()` is better than `ListItem()` or anything similar. What do you think?